App.tsx

```diff
@@ -2,7 +2,6 @@ import React, { useCallback } from 'react';
 import { StyleSheet, View } from 'react-native';
 import { useFonts } from 'expo-font';
 import * as SplashScreen from 'expo-splash-screen';
-import { Colors } from '@styles/colors';
```

Comment:
Return this import because it is used to replace color literals
```suggestion
import { Colors } from '@styles/colors';
```